int32 getpose_start
---
float64[] pose_at